crates/api_models/src/enums.rs

```diff
@@ -195,6 +196,7 @@ pub enum RoutableConnectors {
     Adyen,
     Airwallex,
     Authorizedotnet,
+    // Bankofamerica, Added as template code for future usage
     Bitpay,
     Bambora,
```

Comment:
Can you please put them in the alphabetical order?

Reply:
Done